.github/workflows/e2e.yaml

```
@@ -21,7 +26,13 @@ jobs:
         run: uv python install 3.12
 
       - name: Install jumpstarter
-        run: uv tool install "git+https://github.com/jumpstarter-dev/jumpstarter.git@shell-command"
+        run: |
+          if [ -z "${{ inputs.rev }}" ]; then
+            REV="shell-command"
```

```suggestion
            REV="main"
```
Just triggered merge on the other repo :-)